code/game/objects/items/weapons/null_rod.dm

```diff
@@ -0,0 +1,73 @@
+/obj/item/weapon/nullrod
+	name = "null rod"
+	desc = "A rod of pure obsidian, its very presence disrupts and dampens the powers of paranormal phenomenae."
+	icon_state = "nullrod"
+	item_state = "nullrod"
+	flags = FPRINT
+	slot_flags = SLOT_BELT
+	force = 15
+	throw_speed = 1
+	throw_range = 4
+	throwforce = 10
+	w_class = 1
+
+/obj/item/weapon/nullrod/suicide_act(mob/user)
+		viewers(user) << "<span class='danger'>[user] is impaling \himself with \the [src]! It looks like \he's trying to commit suicide.</span>"
```

Comment:
Unless github is drunk the identation is fucked up.

Also, `visible_message()`